iroha_python/ws.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+import json
+import iroha2
+
+from iroha2 import Client
+from iroha2.data_model.events import EventFilter, pipeline
+
+
+if __name__ == "__main__":
+    cfg = json.loads(open("./config.json").read())
+    cl = Client(cfg)
+    filter = EventFilter.Pipeline(pipeline.EventFilter(
+        entity=pipeline.EntityType.Transaction(),
+        hash=None,
+    ))
+    listener = cl.listen(filter)
+    while True:
+        # Strange thing -- ws doesnt work as an iterator
```

Comment:
I think we need to fix it, maybe you forgot to add `__iter__` or something else? Please consult with Ruslan on this.

Reply:
The thing is that pyo3 is working weirdly. I guess we can wrap it with python class on python side.